docs/Plugins/github.md

```diff
@@ -12,6 +12,9 @@ This plugin collects GitHub data through [REST API](https://docs.github.com/en/r
 - How fast do we respond to customer requirements?
 - Was our quality improved or not?
 
+## Supported Versions
+Available for all versions
```

Comment:
Should it be GitHub Cloud? DevLake does not support GitHub Enterprise for now.

Reply:
Sure, I have updated.